android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -76,7 +77,6 @@ public ScreenStackHeaderConfig(Context context) {
     if (context.getTheme().resolveAttribute(android.R.attr.colorPrimary, tv, true)) {
       mToolbar.setBackgroundColor(tv.data);
     }
-    mToolbar.setClipChildren(false);
```

Comment:
Is there a reason why this was removed?

Reply:
Probably wrongly rebased to the current master.